packages/components/src/mobile/global-styles-context/utils.native.js

```diff
@@ -104,7 +104,7 @@ export function getBlockColors(
 
 	// Custom colors.
 	Object.entries( blockStyleAttributes ).forEach( ( [ key, value ] ) => {
-		const isCustomColor = startsWith( value, '#' );
+		const isCustomColor = value?.startsWith?.( '#' );
```

Comment:
Curious why we need `?.` for `startsWith`.. Can the value be something else except `string`?

Reply:
There were a few failing tests that you can reproduce if you revert this change and run:

```
npm run native test packages/components
```